wettercom/README.md

```diff
@@ -387,6 +426,7 @@ wetter:
                     type: num
 
                 wind:
+                    type: foo
```

Comment:
Do we need all that `type: foo` stuff?

Reply:
Not really. It was included by me at a time when shng mocked about items without a type at startup.

Reply:
shNG sets the type for items to foo, if no type specification is given. I think it is implemented that way since v1.3

Reply:
cleaned up the example yaml